src/components/form/fields/passwordField.jsx

```diff
@@ -106,47 +108,53 @@ class PasswordField extends React.Component {
                         ? <HidePasswordIcon onClick={this.toggleShowPassword} />
                         : <ShowPasswordIcon onClick={this.toggleShowPassword} />)}
                 </div>
-                {this.props.isTouched && <PasswordStrength score={this.props.strength || 0} />}
+                {isTouched && <PasswordStrength score={this.props.strength || 0} />}
                 {validation.error && <FormError>{validation.error}</FormError>}
+                {isTouched && <PasswordPolicyRules value={value} rules={this.props.enabledRules} />}
             </div>
         </FormGroupContainer>;
     }
 }
 
-function checkSpecialsCharacters(password, passwordPolicy) {
-    if (!!passwordPolicy.specialCharacters) {
-        const SPECIAL_CHARACTERS = " !\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~"
-        const passwordChars = Array.from(password)
-        return !passwordChars.find(c => SPECIAL_CHARACTERS.includes(c))
-    } else {
-        return false
-    }
-}
+function listEnabledRules(i18n, passwordPolicy) {
+    if (!passwordPolicy) return {};
+
+    const SPECIAL_CHARACTERS = " !\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~";
+
+    const rules = {
+        minLength: {
+            label: i18n('validation.password.minLength', { min: passwordPolicy.minLength }),
```

Comment:
I think `validation.password` can be stored in a common variable.

Reply:
Also those rules should be declared outside of the method, otherwise we'll create those variables every time we call this function.

Reply:
I don't see why you want to store `validation.password`, it's the key used for the i18n message.

Reply:
The `listEnabledRules` method is only called at the initialization of the component, so it shouldn't be a problem.